src/Facade/Services/ISernosNoteService.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Linn.Products.Facade.Services
+{
+    using System.Collections.Generic;
+
+    using Linn.Common.Facade;
+    using Linn.Products.Domain.Linnapps;
+    using Linn.Products.Resources;
+
+    public interface ISernosNoteService
+    {
+        IResult<SernosNote> GetSernosNoteById(int sernosNoteId);
+
+        IResult<SernosNote> GetSernosNote(string sernosGroup, int? sernosNumber, string transCode);
```

Comment:
sernos number is optional?

Reply:
Yeah in the DB only the note id and note fields are required, so I was trying to match that in the domain. I agree that it probably doesn't make sense in the service here. 

Just realised I'm actually not using this method in the service so I could remove it entirely.